packages/teams-js/src/public/authentication.ts

```diff
@@ -466,6 +466,8 @@ export namespace authentication {
   /**
    * @deprecated
    * As of 2.0.0-beta.1.
```

Comment:
Sorry! Just realized this--the @deprecated tag generates a full sentence at publish time: [This API is now deprecated](https://docs.microsoft.com/en-us/javascript/api/@microsoft/teams-js/authentication.legacycallbacks?view=msteams-client-js-beta)
. . . so it kind of strands "As of 2.0.0.-beta.1." on its own line without much context (see link above for example). Could we rework each instance of "As of 2.0.0.-beta.1." in this file? Also @erinha correct me if I'm wrong, but I think we landed on dropping the "-beta.1" from the deprecated version note. 

How about something like: **As of 2.0.0, this [function|interface] has been deprecated in favor of a Promise-based model.**

Reply:
Got it, I'll make that change

Reply:
Awesome, thanks! Everything else looks good.

Reply:
@erikadoyle if you are able to approve this PR, please feel free to do that when you're happy with the updated comments!